.github/workflows/release-publish.yaml

```diff
@@ -4,7 +4,7 @@ on:
     tags:
       # These tags should be protected, remember to enable the rule:
       # https://github.com/canonical/starbase/settings/tag_protection
-      - "[0-9]+.[0-9]+.[0-9]+"
+      - '[0-9]+\.[0-9]+\.[0-9]+'
```

Comment:
how come?

Reply:
Yeah that came from a change I made during experimentation that didn't turn out to be necessary. Undone!